drafts/SC2-4-4+SC4-1-2-link-names.md

```diff
@@ -0,0 +1,147 @@
+---
+rule_id: SC2-4-4+SC4-1-2-link-names
```

Comment:
How come it's link name and not link text?  Curious rather than suggesting a change.